.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  release:
```

Comment:
It should create release for tag push on master branch only. So https://github.community/t/run-workflow-on-push-tag-on-specific-branch/17519

Reply:
Nice catch. Will fix it